.github/workflows/pull.yml

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# A workflow to send CI-related helpful information to PRs
+name: pull
+on:
+  pull_request:
+
+run-name: CI status hello ${{ github.event.pull_request.number }} - ${{ github.event.pull_request.head.sha }}

```

Comment:
dumb question, what does this guy do?

Reply:
It's nothing special, just a title of the github action, e.g. https://github.com/NVIDIA/Fuser/actions/runs/11602203436/job/32306657201?pr=3315